### PR TITLE
Adds migration job

### PR DESCRIPTION
**Pull request by sm43**

### CODE REVIEW

File: api/pipeline.yaml [hunk 1]
```diff
@@ -91,7 +124,7 @@ spec:
               printf no | tee /tekton/results/output-result
             fi
       runAfter:
-        - build-push-image
+        - create-new-migration-job
```
Comment (vinamra28) [inline api/pipeline.yaml]: We will be needing both. What if the buildah task for API image build fails?

Comment (sm43) [reply api/pipeline.yaml]: okay

Comment (sm43) [reply api/pipeline.yaml]: updated

File: api/pipeline.yaml [hunk 2]
```diff
@@ -71,13 +72,45 @@ spec:
...
+          value: ./db.Dockerfile
+      runAfter:
+        - unit-test
+    - name: create-new-migration-job
+      taskRef:
+        name: kubernetes-actions
+      params:
+        - name: script
+          value: |
+            kubectl delete job db-migration --ignore-not-found
```
Comment (vinamra28) [inline api/pipeline.yaml:105]:
```suggestion
            kubectl create namespace $(params.HUB_NAMESPACE)
            kubectl delete job db-migration -n $(params.HUB_NAMESPACE)  --ignore-not-found
```

**Pull request merged**